src/Lykke.Job.HistoryExportBuilder/HistoryConverter.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using Lykke.Job.HistoryExportBuilder.Core.Domain;
+using Lykke.Service.History.Contracts.Enums;
+using Lykke.Service.History.Contracts.History;
+using Lykke.Service.OperationsHistory.AutorestClient.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Lykke.Job.HistoryExportBuilder
+{
+    public static class HistoryOperationToResponseConverter
+    {
+        public static IEnumerable<HistoryModel> ToHistoryModel(this BaseHistoryModel baseModel)
+        {
+            switch (baseModel)
+            {
+                case CashinModel cashin:
+                    yield return cashin.ToHistoryModel();
```

Comment:
why yield here? you can return cashin.ToHistoryModel() and method should return HistoryModel instead of IEnumerable